### ISSUE TITLE
Compatibility with Symfony 4

### ISSUE THREAD

**Issue by daaru00** [flint/Lurker#30]
Hi,

I cannot require this package, I'm using Symfony v4.0.4 but it require `"symfony/event-dispatcher" : "^2.2|^3.0"`.

So this raise an error:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: remove symfony/symfony v4.0.4
    - Conclusion: don't install symfony/event-dispatcher v3.4.4
    - Installation request for henrikbjorn/lurker ^1.2 -> satisfiable by henrikbjorn/lurker[1.2.0].
    - Conclusion: don't install symfony/symfony v4.0.4
    - henrikbjorn/lurker 1.2.0 requires symfony/event-dispatcher ^2.2|^3.0 -> satisfiable by symfony/event-dispatcher[.....]
```

**Comment by greg-1-anderson** [comment]
See https://github.com/flint/Lurker/pull/29

**Comment by cedx** [comment]
Same issue here because I'm using v2.0.0 of Robo:

```shell
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Installation request for consolidation/robo ^2.0.0 -> satisfiable by consolidation/robo[2.0.0].
    - Conclusion: remove symfony/event-dispatcher v3.4.33
    - henrikbjorn/lurker 1.2.0 requires symfony/event-dispatcher ^2.2|^3.0 -> satisfiable by symfony/event-dispatcher[v3.4.33, v2.2.0, ..., v3.4.9].
    - henrikbjorn/lurker 1.2.0 requires symfony/event-dispatcher ^2.2|^3.0 -> satisfiable by symfony/event-dispatcher[v3.4.33, v2.2.0, ..., v3.4.9].
    - symfony/event-dispatcher v2.5.0 conflicts with symfony/console[v4.3.5].
    - [...]
    - symfony/event-dispatcher v2.4.9 conflicts with symfony/console[v4.3.5].
    - consolidation/robo 2.0.0 requires symfony/console ^4.3.5 -> satisfiable by symfony/console[v4.3.5, v4.3.6].
    - Conclusion: don't install symfony/console v4.3.6
    - Installation request for henrikbjorn/lurker ^1.2.0 -> satisfiable by henrikbjorn/lurker[1.2.0].
```

**Comment by pixelbrackets** [comment]
Last release of this package was 2016-03-16. The dependencies don't allow using Symfony 4.

Solution: Use the drop-in replacement package [totten/lurkerlite](https://packagist.org/packages/totten/lurkerlite) instead.

```bash
composer require totten/lurkerlite
```

This removes »henrikbjorn/lurker« automatically and installs the replacement package.
